server/src/middleware/__generated__/swagger.json

```diff
@@ -553,7 +577,7 @@
 					},
 					"nextCursor": {
 						"type": "string",
-						"description": "Needed for firestore operations which do not support offset\nbased pagination\n\n**Will be undefined in case of last page**"
+						"description": "Needed for firestore operations which do not support offset\r\nbased pagination\r\n\r\n**Will be undefined in case of last page**"
```

Comment:
Bruh it's doing this again, I'll try investigate why it does this but for now just need to remove all the `\r` (it's a Linux vs windows moment)